src/styles/blocks/gallery.scss

```diff
@@ -0,0 +1,109 @@
+.gallery {
+  &__content {
+    display: grid;
+    grid-template-columns: repeat($column-count-mobile, 1fr);
+    column-gap: 20px;
+    &--special {
+      grid-template-columns: repeat(3, 1fr);
+    }
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat($column-table-mobile, 1fr);
+      column-gap: 24px;
+    }
+    @include onDesktop {
+      display: grid;
+      grid-template-columns: repeat($column-table-desktop, 1fr);
+      column-gap: 24px;
+    }
+  }
+  &__photo-big {
+    @include onTablet {
+      grid-column: 1/5;
+    }
+    @include onDesktop {
+      grid-column: 1/8;
+    }
+  }
+  &__photo-big-reverse {
+    @include onTablet {
+      grid-column: 4/9;
+    }
+    @include onDesktop {
+      grid-column: 4/13;
+    }
+  }
+  &__photo-small {
+    @include onTablet {
+      grid-column: 5/9;
+    }
+    @include onDesktop {
+      grid-column: 8/13;
+    }
+  }
+  &__photo-small-reverse {
+    @include onTablet {
+      grid-column: 1/4;
+    }
+    @include onDesktop {
+      grid-column: 1/4;
+    }
+  }
+  &__photo-one {
+    @include onTablet {
+      grid-column: 1/3;
+    }
+    @include onDesktop {
+      grid-column: 1/5;
+    }
+  }
+  &__photo-two {
+    @include onTablet {
+      grid-column: 3/5;
+    }
+    @include onDesktop {
+      grid-column: 5/9;
+    }
+  }
+
+  &__photo-tree {
+    @include onTablet {
+      grid-column: 5/7;
+    }
+    @include onDesktop {
+      grid-column: 9/13;
+    }
+  }
+  &__photos {
+    padding-bottom: 48px;
+  }
+  &__photo-container {
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 256px;
+    overflow: hidden;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+    @include onDesktop {
+      height: 256px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__image {
```

Comment:
don't cut the head of statue 😄 

<img width="1157" alt="image" src="https://user-images.githubusercontent.com/99652389/181455684-08c8436c-d87c-4084-b335-a8a6d24322d5.png">

Reply:
<img width="1367" alt="image" src="https://user-images.githubusercontent.com/99652389/181455855-a895a257-b192-4d51-8b0c-e2f1a6420b82.png">